test/benchmarks/driver_bench/src/suites/multi_bench/grid_fs_upload.mts

```diff
@@ -0,0 +1,40 @@
+import { Readable } from 'node:stream';
+import { pipeline } from 'node:stream/promises';
+
+import { driver, type mongodb } from '../../driver.mjs';
+
+export const taskSize = 52.43;
+
+let bucket: mongodb.GridFSBucket;
+let uploadStream: mongodb.GridFSBucketWriteStream;
+let bin: Uint8Array;
+
+export async function before() {
+  bin = await driver.load('single_and_multi_document/gridfs_large.bin', 'buffer');
+}
+
+export async function beforeEach() {
+  await driver.drop();
+  await driver.create();
+
+  bucket = driver.bucket;
+
+  await bucket.drop().catch(() => null);
+
+  uploadStream = bucket.openUploadStream('gridfstest');
+
+  // Create the bucket.
+  const stream = bucket.openUploadStream('setup-file.txt');
+  const oneByteFile = Readable.from('a');
+  await pipeline(oneByteFile, stream);
```

Comment:
same comment here as for the download tests - can we reuse this for all tests?

Reply:
(except the actual upload stream creation)

Reply:
done